R/arx_classifier.R

```diff
@@ -171,11 +190,25 @@ arx_class_epi_workflow <- function(
         )
     }
   }
-  o2 <- rlang::sym(paste0("ahead_", args_list$ahead, "_", o))
+  ahead_out_name <- glue::glue("ahead_[0-9]*_{pre_out_name}")
+  method_adjust_latency <- args_list$adjust_latency
+  if (method_adjust_latency != "none") {
+    # only extend_ahead is supported atm
```

Comment:
Stale?
```suggestion
```

Reply:
probably? I've added an issue https://github.com/cmu-delphi/epipredict/issues/389 to explicitly add tests

Reply:
Then other methods should error?

Reply:
I think they probably work; is resolving whether they do or not blocking?

Reply:
As a reviewer, I see a comment here that doesn't match the surrounding code. If they work, then let's remove the comment. If we don't know if they work, or don't think they work, then let's put an error here that catches if the user asks for them and refuses to procede.

Reply:
got it, comment removed and moved to an issue

Reply:
Alright, explicitly tested it and added errors b/c it wasn't working. Then again, the user would've hit the errors I got anyway, so this is just adding a more explicit wrapper patch.